.project-keeper.yml

```diff
@@ -9,3 +9,4 @@ linkReplacements:
 excludes:
   - "E-PK-CORE-18: Outdated content: '.github/workflows/ci-build.yml'"
   - "E-PK-CORE-18: Outdated content: '.github/workflows/release_droid_upload_github_release_assets.yml'"
+  - "W-PK-CORE-153: Project-keeper version 4.1.0 is outdated. Please update project-keeper to latest version 4.2.0."
```

Comment:
Is there a specific reason why you can't use the latest PK?
If so: Could you please add a comment to explain the reason?

Reply:
As discussed with @kaklakariada , Running error-code-crawler in it's own repo requires customizing the ci-build (first install, then run plugin). I added the comment to this exclusion